Dockerfile

```diff
@@ -78,6 +78,13 @@ RUN echo $DEV_USER:$DEV_USER_PASSWD | chpasswd
 # Initialize development environment for $DEV_USER.
 RUN sudo -u $DEV_USER -H git config --global credential.helper 'cache --timeout=3600'
 
+COPY entrypoint /
+
+RUN chmod 0755 /entrypoint \
+ && sed "s/\$DOCKER_USER/$DEV_USER/g" -i /entrypoint
```

Comment:
why not just check in the file with `0o755` and the proper content. Then you avoid this extra layer in the container image.

Reply:
I will change the source permission and drop the `chmod` call but the user name change in place with `sed` is to use the same user name as defined in `ARG DEV_USER=builder` otherwise can be forgotten when DEV_USER change.

Reply:
make sense, just move your change up a few lines so that you can do a single `RUN` that setup the git helper cache *and* does this.

Reply:
joined with the `RUN` of the `chpasswd` a few lines before git helper cache.